frontend/src/apiAccess/helpers.js

```diff
@@ -8,9 +8,14 @@ export function getNextPageNumberFromUrl({ url }) {
 }
 
 const _toCamel = (s) => {
-  return s.replace(/([-_][a-z])/gi, ($1) => {
-    return $1.toUpperCase().replace("-", "").replace("_", "");
-  });
+  let arr = s.split("");
```

Comment:
This kind of change would have been more easily done by following TDD. Can you please write tests for it?  Also, TDD is a really really good tool to use when bug fixing.

Step 1: write a test that fails because of the bug (red)
Step 2: fix the bug (green)
Step 3: refactor